TRIAGING.md

```diff
@@ -0,0 +1,186 @@
+
+Please note that there are also automation policies in place. For example, if an issue is labeled as https://github.com/conda/infra/labels/pending%3A%3Afeedback and https://github.com/conda/infra/labels/unreproducible, that issue will be auto-closed after a month of inactivity.
+
+
+### How are new labels defined?
+
+Labels are defined using a scoped syntax with an optional high-level category (_e.g._, source, tag, type, etc.) and a specific topic, much like the following:
+
+- `[topic]`
+- `[category::topic]`
+- `[category::topic-phrase]`
+
+This syntax helps with triage enforcement; at minimum, both `type` and `source` labels should be specified on each issue before moving it from "`Triaging`" to "`Ready`".
+
+There are a number of labels that have been defined for the different `conda` projects. In order to create a streamlined triaging process, label terminologies are standardized using similar (if not the same) labels.
+
+
+### How are new labels added?
+
+New **global** labels (_i.e._, generic labels that apply equally to all `conda` repos) can be added to the `conda/infra`'s [`.github/global.yml` file](https://github.com/conda/infra/blob/main/.github/global.yml); new **local** labels (_i.e._, labels specific to particular `conda` repos) can be added to each repository's `.github/labels.yml` file (_e.g._, `conda/conda`'s [label file](https://github.com/conda/conda/blob/master/.github/labels.yml)). All new labels should follow the labeling syntax described in the "How are new labels defined?" section of this document.
+
+
+### Are there any templates to use as responses for commonly-seen issues?
+
+Some of the same types of issues appear regularly (_e.g._, issues that are duplicates of others, tickets that should be filed in the Anaconda issue tracker, errors that are due to a user's specific setup/environment, etc.).
+
+Below are some boilerplate responses for the most commonly-seen triaging issues:
+
+<details>
+<summary><b>Duplicate Issue</b></summary>
+
+<pre>
+
+This is a duplicate of <b>[link to primary issue]</b>; please feel free to continue the discussion there.
+
+</pre>
```

Comment:
Since the trailing newline (after the text but before `</pre>`) causes a weird extra newline let's remove both the leading & trailing. We want this:

<pre>
Example
</pre>

instead of this:

<pre>

Example

</pre>